scripts/bump-candidate.sh

```diff
@@ -6,8 +6,8 @@
 # a verdaccio instance that serves local tarballs, and if those tarballs have the same version as
 # already published modules, it messes things up.
 #
-# It does so by using a pre-release rc tag, making it so that locally packed versions will always be
-# suffixed with '-rc', distinguishing it from publisehd modules.
+# It does so by using a pre-release "test" tag, making it so that locally packed versions will always be
+# suffixed with '-test', distinguishing it from published modules.
```

Comment:
⚠️ This prerelease identifier is **not** supported by `jsii-pacmak`.

Reply:
Good point. Initially I thought these tests are running before `pacmak`, but I think you are right. Let me check.

Reply:
Okay, fixed.